EntityFramework.Exceptions.Tests/OracleTests.cs

```diff
@@ -10,6 +11,14 @@ public class OracleTests : DatabaseTests, IClassFixture<OracleTestContextFixture
     public OracleTests(OracleTestContextFixture fixture) : base(fixture.DemoContext)
     {
     }
+    
+#if BULK_OPERATIONS
+    [Fact(Skip = "Skipping until ORA-01407 is supported")]
+    public override Task RequiredColumnViolationThrowsCannotInsertNullExceptionThroughExecuteUpdate()
+    {
+        return Task.CompletedTask;
```

Comment:
Is this same as `ORA-01400` but for update?

Reply:
Yes. I intend to add support for it in a new PR after this one is merged.

https://docs.oracle.com/en/error-help/db/ora-01407